.travis.yml

```diff
@@ -17,11 +17,12 @@ matrix:
     - env: PGVERSION=10
     - env: PGVERSION=11
     - env: PGVERSION=12
+    - env: LINTING=true
 before_install:
   - git clone -b v0.7.12 --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
-  - setup_apt
-  - nuke_pg
+  - '[ "$LINTING" == true ] || setup_apt
+  - '[ "$LINTING" == true ] || nuke_pg
```

Comment:
That reads strange. If listing is true then, oh wait, no, if it's not true then. Can we use `&&` instead, or at least add comments about why we need to use this convoluted form? Also I'm not familiar with the first single quote in the line but that might just be the YAML support at Travis to escape to the shell?

Reply:
It was because all the lines need to return true for Travis to pass CI and that doesn't work when you do `&&`. I changed it to `if [ "$LINTING" == "" ]; then setup_apt; fi` now, for better reability.

The quotes are their to indeed just make the whole line a string and not let yaml iterpret characters as things that they are not. (The single quote was simply a mistake on my end, I didn't put one at the end of the line.)

Reply:
I see, thanks for the details! I added some more comments...